### PR TITLE
Use fastcov for faster instead of lcov(reserved).

### PR DESCRIPTION
**Pull request by Shylock-Hg**

### CODE REVIEW

File: Dockerfile.centos [hunk 1]
```diff
@@ -79,6 +81,9 @@ RUN wget -qO- https://github.com/ccache/ccache/releases/download/v3.7.7/ccache-3
...
 
 # Install lcov to /usr/local/bin, require master to make gcc9 happy
-RUN git clone --depth=1 --branch=master --single-branch https://github.com/linux-test-project/lcov.git \
-  && cd lcov && git checkout 40580cd65909bc8324ae09b36bca2e178652ff3f \
+RUN git clone --branch=master --single-branch https://github.com/linux-test-project/lcov.git \
+  && cd lcov && git checkout 75fbae1cfc5027f818a0bb865bf6f96fab3202da \
   && make install && cd ../ && rm -rf lcov
+
+# fastcov
+RUN pip3 install --no-cache-dir fastcov
```
Comment (laura-ding) [inline Dockerfile.centos]: This individual open-source has not been very widely used, it should be more verification？

Comment (Shylock-Hg) [reply Dockerfile.centos]: Yes, I had the same problem before. But seems no better way to speed up this now.

**Comment by dangleptr** [comment]
Is it ready now?

**Comment by Shylock-Hg** [comment]
> Is it ready now?

I think so.

**Comment by dangleptr** [comment]
> > Is it ready now?
> 
> I think so.

Well done.  Could we go ahead with fastcov?  @dutor @laura-ding @yixinglu @bright-starry-sky 

WDYT?

**Pull request merged**